lib/GtkPatch/AboutDialog.vala

```diff
@@ -246,91 +242,90 @@ public class Granite.GtkPatch.AboutDialog : Gtk.Dialog
+        var content_right_box = new Gtk.Box (Gtk.Orientation.VERTICAL, 0);
+        var content_scrolled = new Gtk.ScrolledWindow (null, new Gtk.Adjustment (0, 0, 100, 1, 10, 0));
+        var content_scrolled_vbox = new Gtk.Box (Gtk.Orientation.VERTICAL, 0);
+        var title_vbox = new Gtk.Box (Gtk.Orientation.VERTICAL, 0);
+        var logo_vbox = new Gtk.Box (Gtk.Orientation.VERTICAL, 0);
 
-        content_scrolled.shadow_type = ShadowType.NONE;
-        content_scrolled.hscrollbar_policy = PolicyType.NEVER;
-        content_scrolled.vscrollbar_policy = PolicyType.AUTOMATIC;
+        content_scrolled.shadow_type = Gtk.ShadowType.NONE;
+        content_scrolled.hscrollbar_policy = Gtk.PolicyType.NEVER;
+        content_scrolled.vscrollbar_policy = Gtk.PolicyType.AUTOMATIC;
 
-        content_area.pack_start(content_hbox, true, true, 0);
+        content_area.pack_start (content_hbox, true, true, 0);
 
-        logo_image = new Image();
-        logo_vbox.pack_start(logo_image, false, false, 12);
-        logo_vbox.pack_end(new Box(Orientation.VERTICAL, 0), true, true, 0);
+        logo_image = new Gtk.Image ();
+        logo_vbox.pack_start (logo_image, false, false, 12);
+        logo_vbox.pack_end (new Gtk.Box (Gtk.Orientation.VERTICAL, 0), true, true, 0);
 
         // Adjust sizes
         content_hbox.height_request = 160;
         content_scrolled_vbox.width_request = 288;
 
-        name_label = new Label("");
+        name_label = new Gtk.Label ("");
         name_label.halign = Gtk.Align.START;
-        name_label.set_line_wrap(true);
-        name_label.set_selectable(true);
+        name_label.set_line_wrap (true);
+        name_label.set_selectable (true);
 
         Granite.Widgets.Utils.apply_text_style_to_label (TextStyle.H2, name_label);
 
-        copyright_label = new Label("");
-        copyright_label.set_selectable(true);
+        copyright_label = new Gtk.Label ("");
+        copyright_label.set_selectable (true);
         copyright_label.halign = Gtk.Align.START;
-        copyright_label.set_line_wrap(true);
+        copyright_label.set_line_wrap (true);
 
-        comments_label = new Label("");
-        comments_label.set_selectable(true);
+        comments_label = new Gtk.Label ("");
+        comments_label.set_selectable (true);
         comments_label.halign = Gtk.Align.START;
         comments_label.set_line_wrap(true);
 
-        authors_label = new Label("");
-        authors_label.set_selectable(true);
+        authors_label = new Gtk.Label ("");
+        authors_label.set_selectable (true);
         authors_label.halign = Gtk.Align.START;
-        authors_label.set_line_wrap(true);
+        authors_label.set_line_wrap (true);
 
-        artists_label = new Label("");
-        artists_label.set_selectable(true);
+        artists_label = new Gtk.Label ("");
+        artists_label.set_selectable (true);
         artists_label.halign = Gtk.Align.START;
         artists_label.set_line_wrap(true);
```

Comment:
I was trying to only change lines I was already touching. I'll come back for the other ones, didn't want the diff to be too big